final_prep/annotations.rb

```diff
@@ -3,41 +3,60 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# define a method with Build a Bear attributes
 def build_a_bear(name, age, fur, clothes, special_power)
+  # interpolates name into a string greeting
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # demographics variable is an array of two other variables
   demographics = [name, age]
+  # similar to greeting, interpolate special_power into power_saying string
   power_saying = "Did you know that I can #{special_power}?"
+  # built_bear hash collating all the above variables, cost, and is_cuddly boolean
+  # into a data model
   built_bear = {
+    # map the demographics array to basic_info key
     'basic_info' => demographics,
+    # clothes key accepts array of clothing items
     'clothes' => clothes,
+    # exterior key maps to fur color value
     'exterior' => fur,
+    # cost, a float in USD
     'cost' => 49.99,
+    # key sayings has value of an array collating the strings above and an additional sentence
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  #method returns the hash
   return built_bear
 end
 
+#use method to build bears with the required arguments passed in
```

Comment:
Instead of saying "use method"  you should say "invoke or call method." Solid work on technical language in your annotations but definitely something to keep improving since it's important in interviews.